.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: ruby
 script:
-  - bundle exec rake rubocop
+  - bundle exec rake
```

Comment:
There's no need to tell travis to run that — that's what it does by default: https://docs.travis-ci.com/user/languages/ruby/#Default-Test-Script